netstandard/ref/System.cs

```diff
@@ -4183,9 +4339,13 @@ public abstract partial class Type : System.Reflection.MemberInfo, System.Reflec
         public virtual bool IsSecuritySafeCritical { get { throw null; } }
         public virtual bool IsSecurityTransparent { get { throw null; } }
         public virtual bool IsSerializable { get { throw null; } }
+        public virtual bool IsSignatureType { get { throw null; } }
         public bool IsSpecialName { get { throw null; } }
+        public virtual bool IsSZArray { get { throw null; } }
```

Comment:
Could we have a better name?

Reply:
I remember there were various discussions about this name see https://github.com/dotnet/corefx/issues/16566 but we just couldn't come up with a good name.